server/BudgetTracker.Business/Validation.cs

```diff
@@ -34,8 +34,12 @@ public static async Task<bool> IsAccountRegistrationDuplicate(string username, U
 
         public static bool IsCreateBudgetRequestValid(CreateBudgetRequestContract arguments)
         {
-            bool isValid = arguments.Name != null &&
-                arguments.SetAmount != 0M;
+            bool isValid = arguments.Name != null;
+
+            bool isAmountValid = (arguments.SetAmount == null && arguments.PercentAmount != null && arguments.PercentAmount <= 1.0) ||
```

Comment:
Why can't they set it to 1.0? I mean it doesn't make much sense from a business perspective. Shouldn't we have the same logic for setting the SetAmount to equal the parent's SetAmount

Reply:
That's the affirmative check. So they are allowed to set it to 1.0.

Reply:
Ohhh I missed the less than my bad